swagger.yaml

```diff
@@ -4282,6 +4282,8 @@ definitions:
       - 'valid-platform'
       - 'ntp-synced'
       - 'container-images-available'
+      - 'lso-requirements-satisfied'
+      - 'ocs-requirements-satisfied'
```

Comment:
The new mechanism requires all operators to undergo generic host and cluster validation (even if it doesn't validate anything) so these two needed to be added.

Reply:
Ok.

Reply:
The new API allows the operators to implement both: host- and  cluster- level validation. Current OCS validation is implemented as the latter but in the future it could potentially be split into these two types or extended separately for cluster and host.